src/shadowfax.py

```diff
@@ -1,20 +1,23 @@
+"Handles pathfinder queries"
 import asyncio
 from collections import defaultdict
 import copy
-import uuid
 import hashlib
 import os
-import networkx
-import httpx
 
+import httpx
+import networkx
 from reasoner_pydantic import Message
 
 from src.pathfinder.get_cooccurrence import get_the_curies, get_the_pmids
```

Comment:
I think we want to make these two methods async methods. Otherwise we may be blockif if there are issues in the redis server. If this is too difficult in the short term we want to make an issue and track this.

Reply:
Issue made. I'll explore making these changes in the near future.